templates/cluster_resources.j2

```diff
@@ -77,6 +77,22 @@ primitive rsc_ip_{{ sid }}_ERS{{ ers_instance }} IPaddr2 \
 
 {%- endif %}
 
+{%- if data.monitoring_enabled %}
```

Comment:
I would set `false` by default to this option, so we don't force the user to put it. In fact, having an exporter is optional

Reply:
Hmm, I don't follow. There is no default here (which means the expression will evaluate to false unless the pillar is set to non-false value).

The default is set in SUSE/ha-sap-terraform-deployments#406.

Reply:
Are you sure about that? I'm not totally. I think you would have an error like this if the `monitoring_enabled` is not in the `data` dictionary
```
Unable to manage file: Jinja variable 'salt.utils.odict.OrderedDict object' has no attribute 'monitoring_enabled'
```

What I mean is that I would put this as `{%- if data.monitoring_enabled|default(false) %}`, to make this parameter optional. Otherwise, we are forcing to have `true` or `false` (or something at least) in this entry. At this moment the parameter is not optional anymore

Reply:
Ohh, I see what you mean, now. So, the `|ḍefault` filter also acts as a null coalesce operation? Yeah, I can add that no-problem.